src/lib/hooks/exported.ts

```diff
@@ -49,22 +50,29 @@ export interface FailedUploadStatus extends BaseUploadStatus {
   status: 'failed';
 }
 
-export interface CreateUploadStatus extends BaseUploadStatus {
-  status: 'created';
+interface BaseSuccessUploadStatus extends BaseUploadStatus {
+  /**
+   * Upon upload, there a validation to determine if the page content is renderable or not.
+   * This is the result of that validation.
+   *
+   * - `status`: A flag for if the resource is renderable or not.
+   * - `error`: The rendering error. This is only present if the validation failed.
+   * - `message`: Additional details about the rendering error. This is only present if the validation failed.
```

Comment:
would be rad if these could be derived from the `json-schema-to-ts` type, but alas